locale/forum-statistics-widget.yml

```diff
@@ -0,0 +1,14 @@
+fof-forum-statistics-widget:
+  forum:
+    navbar:
+      widget_title: Forum Statistieken
+      discussions_count: "Aantal Discussies: {discussionsCount}"
+      posts_count: "Aantal Posts: {postsCount}"
+      users_count: "Aantal Leden: {usersCount}"
+      latest_member: "Laatst geregistreerd: "
+  admin:
+    settings:
+      title: Widget Settings
+      widget_order_label: Widget Order
+      widget_helper_text: "Edit this setting to adjust this widget's position on the sidenav section of your Flarum forum.
+      The higher the number, the higher this widget will be on the sidenav items list (this value can be either positive or negative)."
```

Comment:
Forgot to translate? 😛 And maybe rename the file to `fof-forum-statistics-widget`

Reply:
Did not find it really necessary  for the admin part but I will translate it. And you are right about the filename. I will change that.